multio/tests/__init__.py

```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```

Comment:
these PEP 263 source code encodings are needed in python 3

Reply:
I removed them.
TBH, I was tempted to use "keep @raylu happy" as the commit message! :P :D

Reply:
About 50% of my commits are for just that reason ^